src/main/java/racing/RacingGame.java

```diff
@@ -0,0 +1,41 @@
+package racing;
+
+
+import racing.strategy.MoveStrategy;
+import racing.strategy.RacingMoveStategy;
+import racing.view.InputView;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class RacingGame {
+
+    private MoveStrategy moveStrategy;
+
+    public static void main(String[] args) {
+        int countOfRacingCar = InputView.getRacingCarInput();
+        int countOfRacing = InputView.getRacingInput();
+
+        MoveStrategy moveStategy = new RacingMoveStategy();
+        RacingGame racingGame = new RacingGame(moveStategy);
+        List<RacingCar> racingCarList = racingGame.setRacing(countOfRacing);
+        List<RacingCar> racingResult = racingGame.startRacing(countOfRacing, racingCarList);
+        System.out.println(racingResult);
+        ResultView.viewRacingResult(racingResult);
+    }
+    public RacingGame() {}
+
+    public RacingGame(MoveStrategy moveStrategy) {
+        this.moveStrategy = moveStrategy;
+    }
+
+    public List<RacingCar> setRacing(int countOfRacingCar) {
+       return Stream.generate(() -> new RacingCar(moveStrategy)).limit(countOfRacingCar).collect(Collectors.toList());
+    }
+
+    public List<RacingCar> startRacing(int countOfRacingCar, List<RacingCar> racingCarList) {
+        racingCarList.forEach(racingCar -> racingCar.startRacing(countOfRacingCar));
+        return racingCarList;
+    }
```

Comment:
```suggestion
    public void startRacing(int countOfRacingCar, List<RacingCar> racingCarList) {
        racingCarList.forEach(racingCar -> racingCar.startRacing(countOfRacingCar));
    }
```
어차피 매개변수로 전달받은 `racingCarList`를 외부에서도 그대로 사용하면 됩니다.

다만 지금 설계가 잘못 된 상태입니다.

`RacingCars` 라는 일급컬렉션 클래스를 만든 후에  `startRacing` `setRacing`의 역할을 위임하는 게 좋을 것 같아요!